src/masonite/routes/Route.py

```diff
@@ -24,7 +24,7 @@ def get(self, url, controller, module_location=None, **options):
         return HTTPRoute(
             url,
             controller,
-            request_method=["get"],
+            request_method=["get", "head"],
```

Comment:
what is this for?

Reply:
A framework should response to all HEAD requests on its defined routes. So whenever you define a route with GET it will actually register a route with HEAD and a route with GET.

Reply:
It's the expected behaviour for web frameworks : https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD